recipe/meta.yaml

```diff
@@ -1,13 +1,14 @@
-{% set version = "1.6.5" %}
+{% set version = "1.7.1" %}
+
 
 package:
     name: django-redis-cache
     version: {{ version }}
 
 source:
     fn: django-redis-cache-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/d/django-redis-cache/django-redis-cache-{{ version }}.tar.gz
```

Comment:
@sannykr, this is outdated, see https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18

Reply:
@isuruf : I don't understand what you are saying. I had to update the package on conda-forge. When I changed the version number and the md5, I got an error about pypi url. That's why I have used the hardcoded url. I was just asking if it is acceptable to use the hardcoded url of the package.

Reply:
@sannykr please read the example that @isuruf pasted above.

Reply:
@isuruf : Thanks for letting me know. I have made the required changes.

Reply:
@isuruf : Doesn't it work for windows? I don't know why it is disabled. 
When I looked at the pypi page, I found that it is OS independent. Should I use noarch: python in the build section?